wire-ios-sync-engine/Tests/Source/UserSession/ZMUserSessionTestsBase.swift

```diff
@@ -129,6 +129,7 @@ class ZMUserSessionTestsBase: MessagingTest {
             application: application,
             cryptoboxMigrationManager: mockCryptoboxMigrationManager,
             coreDataStack: coreDataStack,
+            analyticsSessionConfiguration: nil,
```

Comment:
Is this still needed?

Reply:
Removed it https://github.com/wireapp/wire-ios/commit/649f36222e7cbf3cb6d8018acdd673cec3427b7d